config/201-clusterrole.yaml

```diff
@@ -87,3 +87,10 @@ rules:
   resources:
   - events
   verbs: *everything
+
+# EventTypes admin
+- apiGroups:
```

Comment:
sorry, I thought I had already made this review comment a while ago, but must have failed to hit the submit button :(

Reply:
And this can be done as a follow on, but just a thought.

Reply:
was just fixing merge conflicts, sorry.
Yes, let me talk with Adam so that he explains me what are the aggregated rules, and I can certainly do it as a follow up PR...

Reply:
thanks for the comment!

Reply:
Created an issue to track this https://github.com/knative/eventing-sources/issues/389

Reply:
I'm not sure that an aggregated role makes sense here, at least not just for EventTypes. Normally we use aggregated roles to say things like 'verb all nouns', where verb is read or write, and nouns are multiple object types, such as 'sources', 'addressables', etc. 

As this change adds EventTypes, and that is a single, known CRD, I don't think it would benefit from an aggregated role.